src/components/UserProfileCard.jsx

```diff
@@ -0,0 +1,35 @@
+import { useVideo } from "contexts/contexts";
+
+export default function UserProfileCard() {
+  const { videoData } = useVideo();
+  const { watchList, likedList, playlists } = videoData;
+  console.log(videoData, watchList, likedList, playlists);
+
+  return (
+    <div className="profile-card shadow">
+      <header className="profile-card-header">
+        <h1>Hello, Daniel!</h1>
```

Comment:
Is this meant to be hardcoded?

Reply:
No, that was a miss. Thank you for pointing it out!